modules/azure/build.gradle

```
@@ -2,6 +2,7 @@ description = "Testcontainers :: Azure"
 
 dependencies {
     api project(':testcontainers')
+    api project(':mssqlserver')
```

documentation must mention dependency on this module

```suggestion
    testImplementation project(':mssqlserver')
```

---

Left as-is due to: https://github.com/testcontainers/testcontainers-java/pull/9795#discussion_r1919291557